styleguide/_layouts/default.jade

```diff
@@ -17,9 +17,23 @@ html(lang="en")
     link(href="#{ css }", rel="stylesheet")
     link(href="css/main.css", rel="stylesheet")
     link(href="https://fonts.googleapis.com/css?family=Open+Sans:700,300,200,600,800,400|Khand:700,600,500|Montserrat:400,700", rel="stylesheet")
+    script(type="text/javascript").
+      WebFontConfig = {
+        google: { families: [ 'Khand:700' ] }
+      };
```

Comment:
If we're going to do this, we probably should use this to pull in all google font families, not just Khand:700.  At this point, we're double loading the font.